### ISSUE TITLE
Tests for 0.09 fail with Plack::Component::Tags::HTML 0.1

### ISSUE THREAD

**Issue by andk** [michal-josef-spacek/Plack-App-Tags-HTML#2]
Sample fail report: http://www.cpantesters.org/cpan/report/455d2ef6-a7e7-11ed-bb63-983bf6d2463d

Statistical analysis from my smokers hints that the likely blame candidate is Plack::Component::Tags::HTML 0.1:

``` text
****************************************************************
Regression 'mod:Plack::Component::Tags::HTML'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    9224755853730832.00
[1='eq_0.07']         0.0000          0.0000       0.14
[2='eq_0.08']        -0.0000          0.0000      -0.11
[3='eq_0.09']        -0.0000          0.0000      -0.92
[4='eq_0.1']         -1.0000          0.0000    -4817471627343198.00

R^2= 1.000, N= 77, K= 5
****************************************************************
```

**Comment by michal-josef-spacek** [comment]
@andk Yes, that's true. I changed the API of Tags::HTML::Page::Begin and this is the root cause.
I don't know what can I do with an old version.

**Comment by michal-josef-spacek** [comment]
Ahh, issue in module, working on it.

**Comment by michal-josef-spacek** [comment]
Fixed in 0.10
